File: python-lhdiff-project/datasets/Aakanksha/File03_New.py
# File03_New.py - Methods Reordered

class File03:
    def add(self, a, b):  # This method is now first
        return a + b

    def multiply(self, a, b):  # This method is now second
        return a * b

    def display_result(self, result):
        print("Final Result:", result)  # Slight modification

File: java-lhdiff-project/datasets/Aakanksha/File03_New.java
// File03_New.java - Methods Reordered
public class File03 {
    public int add(int a, int b) { // This method is now first
        return a + b;
    }

    public int multiply(int a, int b) { // This method is now second
        return a * b;
    }

    public void displayResult(int result) {
        System.out.println("Final Result: " + result); // Slight modification
    }
}
